modules/index/src/main/java/org/apache/ignite/internal/index/IndexManager.java

```diff
@@ -380,6 +403,50 @@ private SortedIndexDescriptor convert(SortedIndexView indexView) {
         );
     }
 
+    private CompletableFuture<BinaryTuple> toIndexKey(TableIndexView indexView, BinaryRow tableRow) {
+        return tableManager.tableAsyncInternal(indexView.tableId(), false).thenApply(table -> {
+            SchemaDescriptor descriptor = table.schemaView().schema(tableRow.schemaVersion());
```

Comment:
Is it possible that there's no schemes with given version in local registry? What will happen in that case?

Reply:
Now it will be waiting for the latest schema if the given one is not presented in the schemaRegistry